Tasks/DockerCompose/dockercomposeconnection.ts

```diff
@@ -40,6 +40,11 @@ export default class DockerComposeConnection extends ContainerConnection {
             process.env["DOCKER_HOST"] = this.hostUrl;
             process.env["DOCKER_TLS_VERIFY"] = 1;
             process.env["DOCKER_CERT_PATH"] = this.certsDir;
+        } else {
+            let dockerCertPath = process.env["DOCKER_CERT_PATH"];
```

Comment:
Its not necessary that every one will want to make TLS connection to its deamon.  In that case you dont want cert directory. Especially in cases where deamon and client is on same machine.